### PR TITLE
User/katerina/arrt update to 0.1.76

### PR DESCRIPTION
**Pull request by ekmineev**
Replace account domain entry with a dropdown.

### CODE REVIEW

**Comment by mafranc** [comment]
Is this branch based off the already merged one? I see all of the commits of that branch in this PR

**Comment by ekmineev** [comment]
> Is this branch based off the already merged one? I see all of the commits of that branch in this PR
Am I doing it wrong? :) This branch have been synched with the main repo so I only see 4 files changed in the PR itself.

**Comment by mafranc** [comment]
Ah, once the PR is completed it's better to move to a new branch, or rebase the branch to the current master (that would get rid of the already merged commits). In practice it's no big deal, just a bit confusing for review

File: ArrtModel/Model/Settings/ArrAccountSettings.h [hunk 1]
```diff
@@ -35,6 +42,9 @@ class ArrAccountSettings : public QObject
     // return the list of available arr regions
     const std::vector<Region>& getAvailableRegions() const { return m_availableRegions; }
 
+	// return the list of supported Remote Rendering account domains
```
Comment (mafranc) [inline ArrtModel/Model/Settings/ArrAccountSettings.h:45]: this file needs clang format

File: ArrtModel/Model/Settings/ArrAccountSettings.cpp [hunk 2]
```diff
@@ -21,6 +21,19 @@ ArrAccountSettings::ArrAccountSettings(QObject* parent)
         {"UK South", "uksouth.mixedreality.azure.com"},
         {"West Europe", "westeurope.mixedreality.azure.com"},
         {"West US", "westus2.mixedreality.azure.com"}};
+
+    // supported account domains
+    m_supportedAccountDomains = {
+        {"Australia East", "australiaeast.mixedreality.azure.com"},
+        {"East US", "eastus.mixedreality.azure.com"},
+        {"East US 2", "mixedreality.azure.com"},
```
Comment (mafranc) [inline ArrtModel/Model/Settings/ArrAccountSettings.cpp:29]: just making sure this is correct (mixedreality.azure.com with no prefix)

**Review by mafranc — CHANGES_REQUESTED** [review]

File: ArrtModel/Model/Settings/ArrAccountSettings.cpp (hunk 2 above)
Comment (ekmineev) [reply ArrtModel/Model/Settings/ArrAccountSettings.cpp]: this is correct and will correspond to "sts.mixedreality.azure.com"

File: ArrtModel/Model/Settings/ArrAccountSettings.h (hunk 1 above)
Comment (ekmineev) [reply ArrtModel/Model/Settings/ArrAccountSettings.h]: ran clang over my changes.

**Review by ChristopherManthei — APPROVED** [review]

**Review by mafranc — APPROVED** [review]

**Pull request merged**